src/lazy/binary/raw/v1_1/e_expression.rs

```diff
@@ -448,7 +451,7 @@ impl<'top> IntoIterator for BinaryEExpArgGroup<'top> {
 impl<'top> EExpressionArgGroup<'top, BinaryEncoding_1_1> for BinaryEExpArgGroup<'top> {
     type Iterator = BinaryEExpArgGroupIterator<'top>;
 
-    fn encoding(&self) -> ParameterEncoding {
+    fn encoding(&self) -> &ParameterEncoding {
```

Comment:
🗺️ `ParameterEncoding` is no longer `Copy`, so accessors hand out a reference instead of a copy.